src/lib/import-export/tests/export/exporters/default-exporter.test.ts

```diff
@@ -185,9 +205,13 @@ describe( 'DefaultExporter', () => {
 		expect( mockArchiver.file ).toHaveBeenNthCalledWith( 1, '/path/to/site/wp-config.php', {
 			name: 'wp-config.php',
 		} );
-		expect( mockArchiver.file ).toHaveBeenNthCalledWith( 2, '/tmp/studio_export_123/file.sql', {
-			name: 'sql/file.sql',
-		} );
+		expect( mockArchiver.file ).toHaveBeenNthCalledWith(
+			2,
+			'/tmp/studio_export_123/studio-backup-db-export-2023-07-31-12-00-00',
+			{
+				name: 'sql/studio-backup-db-export-2023-07-31-12-00-00',
+			}
```

Comment:
I noticed the SQL exported files now don't have the `sql` extension. This will conflict with the import logic as it expects SQL files to have that extension. This is likely to be an undesired side effect of using `generateBackupFilename`. I'll update it to allow setting an extension.

Reply:
After updating the function `generateBackupFilename`, I realized that the name sanitization removes the dot symbol. Hence, I've left it as is and added the extension directly when setting the filename in the exporter:
https://github.com/Automattic/studio/pull/399/commits/fe0aedbb1ab93b7cbde7a404ce7ea852e85b6406